docs/docs/60-new-docs/50-user-guide/20-how-to-guides/16-verification.md

```diff
@@ -0,0 +1,215 @@
+---
+description: Learn how to verify a Stage after Promotion
+sidebar_label: Verifying Freight in a Stage
+---
+
+# Verifying a Stage after Promotion
+
+After successfully promoting a Freight into a Stage, it is often desired to verify the update was successful. Verification is an optional, user-defined process that qualifies Freight in a Stage. Common examples of verification include:
+* Executing containerized integration test scripts
+* Querying monitoring tools (e.g. DataDog, Prometheus) and measuring application KPIs
+* Performing HTTP requests to internal services or systems and interpreting response as success/fail
+* Running load generation tests
+
+When a Stage with verification acts as an upstream to other Stages, Freight must pass verification in that Stage before it is eligible to be promoted to the downstream Stages in the pipeline.
+
+## Configuring Verification
+
+A Stage can be configured with verification by referencing one or more `AnalysisTemplates` in its `spec.verification` field. In the following example, the `dev` `Stage` references an `AnalysisTemplate` named `integration-test`, which executes after any successful Promotion.
+
+```yaml
+apiVersion: kargo.akuity.io/v1alpha1
+kind: Stage
+metadata:
+  name: dev
+  namespace: guestbook
+spec:
+  # ...
+  verification:
+    analysisTemplates:
+    - name: integration-test
+```
+
+## Configuring AnalysisTemplates
+
+An [AnalysisTemplate](../60-reference-docs/50-analysis-templates.md) defines instructions how to perform verification. It might include running a Kubernetes Job, query monitoring tools, interpreting their result, performing HTTP requests, or any combination of these actions. AnalysisTemplates reside in the same `Project` Namespace of the `Stage` from which they are referenced. The following simple `AnalysisTemplate` example will create a Kubernetes `Job` that runs an alpine container that sleeps for 10 seconds.
```

Comment:
```suggestion
An [AnalysisTemplate](../60-reference-docs/50-analysis-templates.md) defines instructions for how to perform verification. It might include running a Kubernetes `Job`, querying monitoring tools, interpreting their result, performing HTTP requests, or any combination of these actions. `AnalysisTemplate`s reside in the same `Project` namespace as the `Stage` from which they are referenced. The following simple `AnalysisTemplate` example will create a Kubernetes `Job` that runs an alpine container that sleeps for 10 seconds:
```